src/pages/MyMints.tsx

```diff
@@ -3,46 +3,56 @@ import { useRecoilValueLoadable } from 'recoil'
 import Card, { CardControlls } from '../components/Card'
 import {
   charityIdFromItem,
-  ItemWithMetadata,
+  ThingWithMetadata,
   listAThing,
   mediaUriFromItem,
+  priceFromItem,
+  thingStatus,
+  ThingStatus,
+  burnTokensOfThing,
 } from '../state/items'
-import { myItemsSelector } from '../state/myItems'
+import { myItemsState } from '../state/myItems'
 import Loader from '../components/Loadaer'
 import { MintbaseContext } from '../contexts/mintbase'
 
 const ThingCard: React.FC<{
-  item: ItemWithMetadata
-  burn: (item: ItemWithMetadata) => void
-  list: (item: ItemWithMetadata) => void
+  item: ThingWithMetadata
+  burn: (item: ThingWithMetadata) => void
+  list: (item: ThingWithMetadata) => void
 }> = ({ item, burn, list }) => {
-  const charityId = charityIdFromItem(item)
+  const status = thingStatus(item)
 
   return (
     <Card
-      username={item.minter}
-      charityId={charityId ? charityId : '-'}
-      title={item.thing.title}
-      price={{ fraction: 1, token: 'NEAR' }}
+      username={item.thing.tokens[0].minter}
+      title={item.metadata.title}
+      charityId={charityIdFromItem(item)}
+      price={priceFromItem(item) || ''}
       url={mediaUriFromItem(item)}
     >
       <CardControlls>
-        <button
-          onClick={() => {
-            burn(item)
-          }}
-          className="uppercase px-8 py-2 border border-blue-600 text-blue-600 max-w-max shadow-sm hover:shadow-lg"
-        >
-          Burn
-        </button>
-        <button
-          onClick={() => {
-            list(item)
-          }}
-          className="uppercase px-8 py-2 border border-blue-600 text-blue-600 max-w-max shadow-sm hover:shadow-lg"
-        >
-          List
-        </button>
+        {status !== ThingStatus.sold && (
```

Comment:
I suggest moving these buttons out of the card itself. Let's have a clean Card view and all actions inside the Item page as buttons alongside a card.

Reply:
Ok

Reply:
Will address these changes in the next PR